mapclassify/_classify_API.py

```diff
@@ -0,0 +1,138 @@
+import mapclassify
+
+__author__ = ("Stefanie Lumnitz <stefanie.lumitz@gmail.com>")
+
+
+_classifiers = {
+    'boxplot': mapclassify.BoxPlot,
+    'equalinterval': mapclassify.EqualInterval,
+    'fisherjenks': mapclassify.FisherJenks,
+    'fisherjenkssampled': mapclassify.FisherJenksSampled,
+    'headtailbreaks': mapclassify.HeadTailBreaks,
+    'jenkscaspall': mapclassify.JenksCaspall,
+    'jenkscaspallforced': mapclassify.JenksCaspallForced,
+    'jenkscaspallsampled': mapclassify.JenksCaspallSampled,
+    'maxp': mapclassify.MaxP,
+    'maximumbreaks': mapclassify.MaximumBreaks,
+    'naturalbreaks': mapclassify.NaturalBreaks,
+    'quantiles': mapclassify.Quantiles,
+    'percentiles': mapclassify.Percentiles,
+    'stdmean': mapclassify.StdMean,
+    'userdefined': mapclassify.UserDefined,
+    }
+
+
+def classify(y, scheme, k=5, pct=[1,10,50,90,99,100],
+             pct_sampled=0.10, truncate=True,
+             hinge=1.5, multiples=[-2,-1,1,2], mindiff=0,
+             initial=100, bins=None):
+    """
+    Classify your data with `pysal.mapclassify.classify`
```

Comment:
Shall we use `pysal.mapclassify.classify` or just `mapclassify.classify`. I guess that a lot of people install mapclassify only, so it may be better to document it independently of meta.